lib/install/validate/validator.go

```diff
@@ -295,6 +299,9 @@ func (v *Validator) network(ctx context.Context, input *data.Data, conf *metadat
 			},
 		},
 	})
+	if input.BridgeNetworkName == input.ClientNetworkName {
+		v.NoteIssue(errors.Errorf("bridge network %s is used for client network, please choose different network", input.BridgeNetworkName))
```

Comment:
the bridge network must not be shared with another network role - client also uses %s